bertopic/_bertopic.py

```diff
@@ -1117,7 +1118,11 @@ def visualize_hierarchy(self,
             top_n_topics: Only select the top n most frequent topics
             width: The width of the figure. Only works if orientation is set to 'left'
             height: The height of the figure. Only works if orientation is set to 'bottom'
-
+            optimal_ordering: bool, optional. If True, the linkage matrix will be reordered so that the distance
```

Comment:
Could you remove the `bool, optional.` here? Thus far, BERTopic only mentions the typing when declaring the parameters and not in the documentation itself.